src/dmtest/vdo/dedupe_tests.py

```diff
@@ -0,0 +1,87 @@
+from dmtest.assertions import assert_raises, assert_equal, assert_near
+from dmtest.vdo.utils import standard_stack, standard_vdo
+import dmtest.fs as fs
+import dmtest.process as process
+import dmtest.units as units
+import dmtest.vdo.stats as stats
+import dmtest.test_register as reg
+
+import os
+import re
+import threading
+import logging as log
+import time
+
+#---------------------------------
+
+def make_delta_stats(stats_post, stats_pre):
+    """Recursive function that transforms only these dictionary values which keys correspond to a `mapping` dict."""
```

Comment:
This description seems confusing to me. It doesn't say what sort of transformation is done. It does operate on keys that map to "int" values, not just those that map to dictionaries.
It's also not important that the function is recursive, though it's important that it recurs into dictionaries found in the values of a passed-in dictionary.

Reply:
The function could be improved on for sure, but I thought this was a good first start. In a perfect world we'd want to somehow mimic the subtract function from Statistics.pm in perl/Permabit. We really only want to subtract certain fields (snapshots, timestamps, etc) but since the fields we really care about are ints and subtractable, I wrote it as is.

Reply:
Comment reworked